### ISSUE TITLE
Unable to send emails with attachments from the platform

### ISSUE THREAD

**Issue by klaussilveira** [oroinc/platform#737]
Even though the interface provides the means to, i'm unable to send any attachments with an email from any case. The emails arrive at the destination, but without attachments. Has anyone noticed this issue?

**Comment by vbyndych** [comment]
Hi @klaussilveira,
Unfortunately issue which you've described can have many different sources. Could you provide more details about your email settings, steps you are doing (probably with image example) and log files. It can help to investigate a problem.

Thank you in advance.

**Comment by vbyndych** [comment]
Hi @klaussilveira,
Unfortunately issue which you've described can have many different sources. Could you provide more details about your email settings, steps you are doing (probably with image example) and log files. It can help to investigate a problem.

Thank you in advance.

**Comment by klaussilveira** [comment]
It's quite simple. Just try to attach a file when sending an email. It won't work.

It is related to this change in particular: https://github.com/oroinc/platform/commit/851a01841fd861b1b6b64ee0f5b8db7ad7d91f5a#diff-6ca779226c4ef57021e3ca5801574c36L57

Not all emails have ID's because the listener that ends up persisting them is only called later.

**Comment by vbyndych** [comment]
Yes, it looks like a bug. So it's internal id is: BAP-15247. I'll get back to you as fast as bug will be fixed.
Thank you for reporting the issue! We greatly appreciate all contributions to Oro projects.

**Comment by klaussilveira** [comment]
Internal ID? Where we, the community, can keep track of those issues? Are they synced with GitHub?

**Comment by vbyndych** [comment]
It will be synced as fast as bug will be fixed, so you will know when fix will be publicly available.
